src/Likvido.Test.Api/Base/ConfigurableTestFixture.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Likvido.Test.Api
+{
+    public abstract class ConfigurableTestFixture<TStartup, TDesignContext, TRuntimeContext>
+        : TestFixture<TStartup, TDesignContext, TRuntimeContext>
+        where TStartup : class
+        where TDesignContext : TRuntimeContext
```

Comment:
I think this constraint is too specific 
```suggestion
        where TDesignContext : DbContext
```